geode-core/src/main/java/org/apache/geode/management/internal/operation/RebalanceOperationPerformer.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.management.internal.operation;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.geode.annotations.Experimental;
+import org.apache.geode.cache.Cache;
+import org.apache.geode.management.cli.Result;
+import org.apache.geode.management.internal.cli.commands.RebalanceCommand;
+import org.apache.geode.management.internal.cli.result.model.InfoResultModel;
+import org.apache.geode.management.internal.cli.result.model.ResultModel;
+import org.apache.geode.management.internal.cli.result.model.TabularResultModel;
+import org.apache.geode.management.operation.RebalanceOperation;
+import org.apache.geode.management.runtime.RebalanceResult;
+
+@Experimental
+public class RebalanceOperationPerformer {
+  public static RebalanceResult perform(Cache cache, RebalanceOperation parameters) {
```

Comment:
`perform` has no dependencies on anything but it's parameters, so it should be an appropriate use of the `static` keyword to declare this.

It's also slightly more succinct that it can be registered as `registerOperation(RebalanceOperation.class, RebalanceOperationPerformer::perform);`

If it were not static, we could still register, but would require a few more characters registerOperation(RebalanceOperation.class, new RebalanceOperationPerformer()::perform);`

Reply:
Because we don't actually need an instance of RebalanceOperationPerformer. Since it has no state you can just reference the method like so: RebalanceOperationPerformer::perform